Package.swift

```diff
@@ -60,12 +60,18 @@ let package = Package(
                 .product(name: "NIOConcurrencyHelpers", package: "swift-nio"),
                 .product(name: "Atomics", package: "swift-atomics"),
                 .product(name: "W3CTraceContext", package: "swift-w3c-trace-context"),
-                // OTLP/HTTP exporter -- only when OTLPHTTP trait is enabled.
+                /// NOTE: Using `.when(traits: ["A", "B"])` is supposed to work as an OR, but is currently broken.
+                ///       so we "splat" it into two conditional dependencies. This probuces a build warning about a
```

Comment:
```suggestion
                ///       so we "splat" it into two conditional dependencies. This produces a build warning about a
```

Reply:
😭 Thank you: https://github.com/swift-otel/swift-otel/pull/300/commits/a218e056d5b1db81da0af8416d99d3e0ba12904e.